ContentApp/BusinessLayer/Action Menu/Models/MultipleFilesActionMenuGeneric.swift

```diff
@@ -29,26 +29,26 @@ struct MultipleFilesActionMenuGeneric {
 
         var actions2: [ActionMenu] = []
```

Comment:
```suggestion
        var multiSelectAction: [ActionMenu] = []
```
Give a proper name of variable.

Reply:
Done